applications/longhorn/README.md

```diff
@@ -0,0 +1,54 @@
+# Overview
+
+This image is based on kubernetes:v1.19.9,  add longhorn  to provide persistent-volume. 
+
+Components included in this image:
+
+* Enterprise-grade distributed storage with no single point of failure
+* Incremental snapshot of block storage
+* Backup to secondary storage (NFSv4 or S3-compatible object storage) built on efficient change block detection
+* Recurring snapshot and backup
+* Automated non-disruptive upgrade. You can upgrade the entire Longhorn software stack without disrupting running volumes!
+* Intuitive GUI dashboard
+
+## How to use it
+
+At the first maker sure all the pods status up in longhorn-system namespace.
+
+### storageclass
+
+This images provide  ```longhorn``` as default stotageclass, you can use it easily in PVC.
+
+### UI dashboard
+
+Once Longhorn has been installed in your Kubernetes cluster, you can access the UI dashboard.
+
+1. Get the Longhorn’s external service IP:
+
+​		``` kubectl -n longhorn-system get svc```
+
+​		For Longhorn v0.8.0, the output should look like this, and the `CLUSTER-IP` of the `longhorn-frontend` is used to access the Longhorn UI:
+
+2. Navigate to the IP of `longhorn-frontend` in your browser.
+
```

Comment:
Awesome work~ could you delete the redundant blanks~

Reply:
ok, I will do it soon. thanks.